caproto/_data.py

```diff
@@ -1,17 +1,107 @@
 import time
+from curio.meta import awaitable
```

Comment:
Yeah, we should look into doing this - or an arguably less elegant solution of offering subclassed `awaitable`-alternatives in a separate file.